spatio_flux/processes/dfba.py

```diff
@@ -105,57 +114,85 @@ def update(self, state, interval):
         else:
             # Handle non-optimal solutions if necessary
             # print('Non-optimal solution, skipping update')
-            for substrate, reaction_id in self.config['substrate_update_reactions'].items():
+            for substrate, reaction_id in self.config[
+                "substrate_update_reactions"
+            ].items():
                 substrate_update[substrate] = 0
 
         return {
-            'substrates': substrate_update,
+            "substrates": substrate_update,
         }
 
 
 # register the process
-core.register_process('DynamicFBA', DynamicFBA)
+core.register_process("DynamicFBA", DynamicFBA)
 
 
 # Helper functions to get specs and states
 def dfba_config(
-        model_file='textbook',
-        kinetic_params=None,
-        biomass_reaction='Biomass_Ecoli_core',
-        substrate_update_reactions=None,
-        biomass_identifier='biomass',
-        bounds=None
+    model_file="textbook",
+    kinetic_params=None,
+    biomass_reaction="Biomass_Ecoli_core",
+    substrate_update_reactions=None,
+    biomass_identifier="biomass",  # TODO: How do we differentiate between biomass between models? Do we need to change the biomass identifier to be unique for each model?
```

Comment:
good question. I did not make this compatible with multi-species dFBA yet, so this one assumes one model. so let's put this on the TODOs. Should be pretty easy.